### PR TITLE
Cleanup RetroArch Install

### PR DESCRIPTION
**Pull request by WingofaGriffin**
See https://github.com/dragoonDorise/EmuDeck/issues/561

Haven't touched Gambatte yet, but I'm tempted to. Leaving them as comments for now, but we should probably clean up formatting too.

Also added minivmac while I was there.

Heavy draft and WIP. Opinions and advice are needed.

### CODE REVIEW

**Comment by Godsbane** [comment]
Leave PPSSPP, because standalone doesn't do achievements.

Leave gambatte as the default. We don't want to mess with that.

Leave the override for dolphin, because even though we don't recommend it, it hard crashes under vulkan and we should avoid that.

**Comment by WingofaGriffin** [comment]
Making ready for review as no one in discord cared lmao

File: functions/EmuScripts/emuDeckRetroArch.sh [hunk 1]
```diff
@@ -1100,31 +1100,33 @@ RetroArch_dreamcast_setConfig(){
...
+
+	# RetroArch_setOverride 'saturn.cfg' 'Yabause'  'input_player1_analog_dpad_mode' '"1"'
+	# RetroArch_setOverride 'saturn.cfg' 'YabaSanshiro'  'input_player1_analog_dpad_mode' '"1"'
 	RetroArch_setOverride 'saturn.cfg' 'Kronos'  'input_player1_analog_dpad_mode' '"1"'
 	RetroArch_setOverride 'saturn.cfg' 'Beetle Saturn'  'input_player1_analog_dpad_mode' '"1"'
 	RetroArch_saturn_3DCRTshaderOff
 }
 
 RetroArch_saturn_bezelOn(){
-	RetroArch_setOverride 'saturn.cfg' 'Yabause'  'aspect_ratio_index' '"0"'
```
Comment (Godsbane) [inline functions/EmuScripts/emuDeckRetroArch.sh:1117]: don't comment anything here please. we want to leave it.

**Comment by Godsbane** [comment]
I care, just haven't had time recently.

Please don't remove the saturn config for non-kronos. as mentioned we are going to be leaving beetle as the main emulator. I'd just revert everything saturn to how it was.

I'll tag @dragoonDorise to review in more depth, since gamegear is not something i use. Maybe there's good reason to keep the configs.

**Comment by WingofaGriffin** [comment]
Apologies, I wasn't targeting you, I was referring to me asking the discord in general if they had any strong opinions, which had no responses.

Will revert the Saturn config stuff, though for people who already have it installed, it shouldn't break anything.

Gamegear configs specifically just commented out the Gearsystem configs, as Genesis Plus GX does Gamegear better.

**Comment by WingofaGriffin** [comment]
Did a big update based on default cores from ESDE. Would love a review @dragoonDorise

**Review by dragoonDorise — APPROVED** [review]

**Pull request merged**